containers/README.md

```diff
@@ -0,0 +1,49 @@
+Subscription Manager & Containers
+=================================
+
+This directory contains scripts and configuration files that are necessary for running
+subscription-manager in a containerized environment. This environment could be created
+using podman. Why do we care about running subscription-manager in a containerized
+environment? We believe that it could be useful for testing of subscription-manager.
+We also support running and testing of subscription-manager using vagrant, but running
+subscription-manager in VM could be a bit cumbersome. Running subscription-manager
+using podman could be more flexible for developers too.
+
+Requirements
+------------
+
+To create and use a containerized environment it is necessary to have following packages
+installed in the system:
+
+ * podman
+ * toolbox
+
+   $ sudo dnf install -y podman toolbox
+
+Create a containerized environment
+----------------------------------
+
+To create a containerized environment you can simply run following script:
+
+    $ toolbox create -i quay.io/candlepin/subscription-manager:main -c rhsm-dev
+    $ toolbox enter rhsm-dev
+
+Reviewing PRs
+-------------
+
+Jenkins will build, tag, and push the PR test environment to
+quay.io/candlepin/subscription-manager:PR-XXXX where XXXX is the PR number.
+This is done as a regular part of the jenkins tests as each test is run in
+one of these containers.
+
+This means that for any PR which has had the jenkins tests complete
+(regardless of success or failure), you can run the following to reproduce
+the jenkins test results (for example if the unit tests failed):
+
```

Comment:
I would like to clarify something here. What remote repository is used here? Is it repository of `subscription-manager` or any other?

Reply:
So the quay.io project (at quay.io/candlepin/subscription-manager) will be configured to watch for changes in this repository's main branch and to build images from there. The jenkins job also builds and pushes images to the quay.io container repository for each PR.

Once this PR is merged I can update quay to build off main automatically so there is always a base to work from (could also do for any branch we like, mostly useful for those we maintain).

Reply:
Thanks :+1: